GeoUtils/proj_tools.py

```diff
@@ -4,6 +4,8 @@
 import rasterio as rio
 from rasterio.crs import CRS
 from shapely.geometry.polygon import Polygon
+from shapely.ops import transform
```

Comment:
This is a confusing import given that `transform` elsewhere in the repo commonly refers to an Affine transform. Please consider referencing as shapely.ops.transform instead.

Reply:
Done!